src/appkit/event/mod.rs

```diff
@@ -25,6 +64,16 @@ impl Event {
         Event(unsafe { Id::from_ptr(objc) })
     }
 
+    /// The event's type.
+    ///
+    /// Corresponds to the `type` getter.
+    pub fn event_type(&self) -> EventType {
```

Comment:
Unsure what the naming convention should be here, as `type` is reserved in Rust. `r#type` didn't seem to be a great solution

Reply:
I opt to go with `kind` these days and I feel like I've seen it in other Rust projects, but I'm open to suggestions.

Reply:
`kind` is what I would opt for as well; it just felt odd since were mirroring the AppKit API.

Reply:
Eh, it's fine in my book - cacao is about wrapping the AppKit/UIKit APIs in a comfortable for Rust way. Deviation is totally fine if it means that the end user has an easy time and is "close enough" to the AppKit/UIKit approach.